packages/block-library/src/group/block.json

```diff
@@ -32,7 +32,9 @@
 			"style": true,
 			"width": true
 		},
-		"__experimentalLayout": true
+		"__experimentalLayout": {
+			"allowSwitching": true
```

Comment:
Should we add explicitly what layout types are supported per block? And even have a theme.json value for that?

Reply:
yes, definitely sounds like something that we'll need. Also I wonder how a block could define a layout without allowing the users to change it at all and have this layout styles generated in the backend.